cargo/.cargo/config.toml

```diff
@@ -3,19 +3,19 @@ target = "{{ rust_target }}"
 
 [target.{{ rust_target }}]
 linker = "ldproxy"
-runner = "espflash flash --monitor" # Select this runner for espflash v3.x.x
-rustflags = [ "--cfg",  "espidf_time64"] # Extending time_t for ESP IDF 5: https://github.com/esp-rs/rust/issues/110
+runner = "espflash flash --monitor"
+rustflags = [ "--cfg",  "espidf_time64"]
```

Comment:
Does the included `libc` still require this flag?

Reply:
I thought ESP Rust 1.83 still does not have the default state flipped? If you say it does, I'll remove right away.

Reply:
> I thought ESP Rust 1.83 still does not have the default state flipped?

I haven't checked, let me do some testing/research. Any way, I think its safer to leave it for a few more releases (we even still have 1.82 mark as `latest`)

Reply:
Anyway, `1.83.0` got released with [`libc@0.2.161`](https://github.com/rust-lang/rust/blob/1.83.0/library/std/Cargo.toml#L42C21-L42C28) while https://github.com/rust-lang/libc/pull/3993 was included in [`libc@0.2.162`](https://github.com/rust-lang/libc/releases/tag/0.2.162)